### DISCUSSION TITLE
⭐️ Starstruck Badge Exchange ⭐️

### DISCUSSION THREAD

**Discussion by michalm57** [Schweinepriester/achievement-testarea#35] [Q&A]
Help me get starstruck badge, I will ⭐️star⭐️ your repo back - provide link in the comment please!
https://github.com/michalm57/currency_exchange

Note: You can see who's actually starred your repo (e.g. "returned the favor") by appending /stargazers to the end of your repo's URL. For example: https://github.com/michalm57/currency_exchange

**Comment by pulkitxm** [comment]
star my  [repo](https://github.com/Pulkitxm/Delta-Apna-College), I'll do it for yours
 [https://github.com/Pulkitxm/Delta-Apna-College](https://github.com/Pulkitxm/Delta-Apna-College)
Have a nice Day !

**Reply by AdityaNow** [reply]
Done 😃 (feel free to star any of my repo https://github.com/AdityaNow/NavtorAssignment & https://github.com/AdityaNow/Wikipedia)
Thank you

**Comment by zoelesv** [comment]
Nice repo. I gave it a ⭐️. Check out my repo and help me get my first star-truck as well! https://github.com/zoelesv/Smathchat

**Reply by KamBRJC** [reply]
star back https://github.com/z-nnovation/GitSwitcher

**Comment by ternera** [comment]
Please star and I will return it: https://github.com/ternera/ternera.org

**Comment by Mattral** [comment]
Help me get starstruck badge, I will ⭐️star⭐️ your repo back - provide link in the comment please!
https://github.com/Mattral/NOTE-Best-Practices-for-Computer-Vision

**Comment by ternera** [comment]
Please star my repository and I will return it: https://github.com/ternera/hacker101-ctf

**Comment by peterbaccaro** [comment]
Please give me a star for my repo and help me get ⭐️ starstruck ⭐️!

https://github.com/peterbaccaro/design-patterns-python

Much appreciated. Thanks in advance.

**Reply by AdityaNow** [reply]
Done, please star mine too - https://github.com/AdityaNow/NavtorAssignment/
Thank you

**Comment by KamBRJC** [comment]
done. now give me all your stars https://github.com/z-nnovation/GitSwitcher
